#! /bin/sh /usr/share/dpatch/dpatch-run
## use-snoopy-from-distro.dpatch by Roland Mas <lolando@debian.org>
##
## All lines beginning with `## DP:' are a description of the patch.
## DP: Use the Snoopy class as provided by the libphp-snoopy Debian package
## DP: instead of shipping a local one.

@DPATCH@
diff -urNad gforge~/plugins/cvstracker/bin/post.php gforge/plugins/cvstracker/bin/post.php
--- gforge~/plugins/cvstracker/bin/post.php	2009-07-19 20:57:25.000000000 +0200
+++ gforge/plugins/cvstracker/bin/post.php	2009-07-23 11:21:52.000000000 +0200
@@ -34,7 +34,7 @@
 require dirname(__FILE__).'/../../env.inc.php';
 require_once ($gfconfig.'plugins/cvstracker/config.php');
 require_once ($gfcommon.'common/include/utils.php');
-require_once ($gfplugins.'cvstracker/common/Snoopy.class.php');
+require_once ('/usr/share/php/libphp-snoopy/Snoopy.class.php');
 
 
 if ($cvs_binary_version != "1.12" &&
diff -urNad gforge~/plugins/scmcvs/bin/aclcheck.php gforge/plugins/scmcvs/bin/aclcheck.php
--- gforge~/plugins/scmcvs/bin/aclcheck.php	2009-07-19 20:57:25.000000000 +0200
+++ gforge/plugins/scmcvs/bin/aclcheck.php	2009-07-23 11:21:52.000000000 +0200
@@ -31,7 +31,7 @@
 require $gfcgfile;
 require_once $gfcommon.'include/utils.php';
 require_once $gfconfig.'plugins/scmcvs/config.php';
-require $gfplugins.'scmcvs/common/Snoopy.class.php';
+require '/usr/share/php/libphp-snoopy/Snoopy.class.php';
 
 // Input cleansing
 $env_cvsroot = (string) $_ENV['CVSROOT'];
diff -urNad gforge~/plugins/svntracker/bin/post.php gforge/plugins/svntracker/bin/post.php
--- gforge~/plugins/svntracker/bin/post.php	2009-07-19 20:57:25.000000000 +0200
+++ gforge/plugins/svntracker/bin/post.php	2009-07-23 11:21:52.000000000 +0200
@@ -32,7 +32,7 @@
 
 
 require $gfconfig.'plugins/svntracker/config.php';
-require $gfplugins.'svntracker/common/Snoopy.class.php';
+require '/usr/share/php/libphp-snoopy/Snoopy.class.php';
 
 if ($svn_tracker_debug) {
 	$file = fopen($svn_tracker_debug_file,"a+");	
